lib/types/Color3_spec.lua

```diff
@@ -1,9 +1,29 @@
 local Color3 = import("./Color3")
 
+local function extractColors(color)
+	return {
+		color.r,
+		color.g,
+		color.b
+	}
+end
+
 describe("types.Color3", function()
-	it("should have a constructor", function()
+	it("should have an empty constructor", function()
+		local color = Color3.new()
+
+		assert.not_nil(color)
```

Comment:
We should probably make sure that the result is `{0, 0, 0}` for the empty constructor